.jazzy.yaml

```diff
@@ -3,9 +3,19 @@ author: Amazon Web Services
 github_url: https://github.com/aws-amplify/amplify-ios
 
 exclude:
-- Pods
+  - Pods
 
 clean: true
 skip_undocumented: false
+undocumented_text: ""
+hide_documentation_coverage: true
 sdk: iphone
 theme: apple
+build_tool_arguments:
+  - -workspace
+  - Amplify.xcworkspace
+  - -scheme
+  - Amplify
+
+docset_icon: readme-images/icon.png
```

Comment:
where did this come from? how about `amplify-icon.png` for clarity?

Reply:
This is the favicon from the main amplify docs site (https://docs.amplify.aws), which you can bounce back and forth between as you navigate the main docs and api docs, so including it here would improve consistency.  With that said, I'm not sure this will show as the favicon because it doesn't show up when testing locally and I don't have a readily available test web server to test it on.  It doesn't hurt to include it though.  I used icon.png because that's the traditional name for a page's favicon, but it can be anything.  I'll rename it.